velox/common/tests/BitUtilTest.cpp

```diff
@@ -416,6 +416,43 @@ TEST_F(BitUtilTest, forEachBit) {
   expectedIndex = 3;
   forEachUnsetBit(data, 2, totalBits - 3, calledOnEveryOther);
   EXPECT_EQ(totalBits - 3, expectedIndex);
+
+  auto numEntries = 100;
+  uint32_t size = bits::nwords(numEntries);
```

Comment:
nit: size is used only once, hence, consider inlining: `data(bits::nwords(totalBits))`